release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -602,6 +602,14 @@ For more information about the MCO certificates, see xref:../security/certificat
 [id="ocp-release-notes-machine-management_{context}"]
 === Machine management
 
+[id="ocp-4-19-capi-mapi-migration_{context}"]
+==== Migrating resources between the Cluster API and the Machine API (Technology Preview)
+
+With this release, you can migrate some resources between the Cluster API and the Machine API on supported platforms.
```

Comment:
```suggestion
With this release, you can migrate some resources between the Cluster API and the Machine API on supported platforms as a Technology Preview feature.
```
just wonder if you want to caveat in the text in case someone blasts by the headline